src/main/java/bdStudents/StudentDAO.java

```diff
@@ -0,0 +1,44 @@
+package bdStudents;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import java.util.List;
+public class StudentDAO {
+    private final SessionFactory sessionFactory = HibernateSession.getSessionFactory();
```

Comment:
hard dependency.
could be replaced with a constructor parameter

`public StudentDAO(SessionFactory sessionFactory) {...}`